pkg/config/template.go

```diff
@@ -1355,6 +1360,13 @@ const templateStringCrioRuntimeDisableHostPortMapping = `# disable_hostport_mapp
 
 `
 
+const templateStringCrioRuntimeTimeZone = `# time_zone specifies the timezone configuration for CRI-O.
+# If an empty string is provided, CRI-O will use the timezone of the host machine.
```

Comment:
That's not true, is it? We should also add the complete docs about the behavior.

Reply:
It uses the timezone of the host machine but displays it in UTC

Reply:
@sohankunkerkar let's document that as well

Reply:
Done